nx-X11/programs/Xserver/Imakefile

```diff
@@ -361,6 +363,7 @@ NXAGENTNXLIBS = -L../../../nxcomp/src/.libs  \
 #else
 NXAGENTNXLIBS = -L../../../nxcomp/src/.libs  \
                 -L../../../nxcompshad/src/.libs \
+                -L../../lib/src/.libs \
```

Comment:
> the -L lines seem identical for most, if not all, cases. So can't we make that simpler?

The Imakefiles for Xserver/ will soon be dropped, too. So I am slightly unwilling to introduce too invasive changes into nx-X11/programs/Xserver/Imakefile at this point.

Doing nothing here.